rc/filetype/markdown.kak

```diff
@@ -45,12 +45,12 @@ evaluate-commands %sh{
     awk c cabal clojure coffee cpp crystal css cucumber d diff dockerfile elixir erlang fish
     gas go haml haskell html ini java javascript json julia kak kickstart
     latex lisp lua makefile markdown moon objc ocaml perl pug python ragel
-    ruby rust sass scala scss sh swift toml tupfile typescript yaml sql
+    ruby rust sass scala scss sh swift toml ts tupfile typescript yaml sql
```

Comment:
There are a bunch of other aliases listed by `pygmentize -L`, like "js" for "javascript" or the various flavours of "sh".
Probably we should add the ones that are used in practice. No strong opinion here.

Reply:
The `case` statement now exists and the aliases can be easily added from now. I set out to fix the typescript case and now that is done. 

Thanks for the `pygmentize -L` tip. Let me review and see if we can get some easy wins like js for javascript and so on.

Reply:
@krobelus I did a quick pass and added some common aliases. Also, `scheme` now has inline highlighting -- it was simply not listed earlier.

I debated added the `bash` => `sh` mapping but strictly speaking `bash` is a super-set of `sh` so its not exactly an alias. There are similar concerns with zsh, ksh etc. 

Anyways, if there are some more aliases I think this can be done as part of another PR. I think this is a good start for now...

Reply:
> I did a quick pass and added some common aliases.

Thanks :)

> I debated added the `bash` => `sh` mapping but strictly speaking `bash` is a super-set of `sh` so its not exactly an alias. There are similar concerns with zsh, ksh etc.

IMO we should definitely support the "shell" alias and probably all others too. We also use filetype=sh for all these shells.